main.go

```diff
@@ -21,13 +21,11 @@ import (
 )
 
 var (
-	success = color.New(color.FgGreen)
-	skipped = color.New(color.FgYellow)
-	fail    = color.New(color.FgHiRed)
+	pass = color.New(color.FgGreen)
+	skip = color.New(color.FgYellow)
```

Comment:
I'm unsure that this is a useful refactor.

Reply:
A simple attempt at making the code readable, currently it reads as _bad_ prose mixing word classes

Reply:
Please elaborate on the danger of merge conflicts, I am not sure I understand the reluctance to make changes because of possible merge conflicts

Reply:
In general, I'm not concerned about variable names as bad prose and the change neither helps nor harms the readability of the code for me. It's not a blocker for this PR.

Merge conflicts aren't dangerous so much as they are tech debt. In this case it's negligible, but worth considering when making changes.

Reply:
Proposed change removed